extension/js/common/core/common.ts

```diff
@@ -94,6 +94,26 @@ export class Str {
     return id;
   };
 
+  // splits the string to matches,
+  // each match is extended till the end of the original string
+  public static splitExtended = (str: string, regexp: RegExp): string[] => {
+    const result: string[] = [];
+    for (; ;) {
```

Comment:
Isn't this a `while (true) {` ? I think a `while` is more standard.

Reply:
It's arguable, as `while (true)` involves extra constant, but if you say so -- corrected.